File: Exercícios/Scripts_C#/ScriptsUsandoC#/ScriptsUsandoC#/Desafio_63.cs
public class Desafio_63
{
    public static void Fibonacci()
    {
        long n1 = 0;
        long n2 = 1;
        long n3 = 0;

        Console.Write("INFORME UM NÚMERO PARA VER SUA SEQUÊNCIA FIBONACCI: ");
        long termo = int.Parse(Console.ReadLine().Trim());

        long cont = 0;
        while (cont < termo)
        {
            Console.Write($"{n1} > ");
            n3 = n1 + n2;
            n1 = n2;
            n2 = n3;
            

            cont++;
        }
        Console.WriteLine(" FIM");
    }
}

File: Exercícios/PythonMundo_1_2_3/Desafio_63.py
n1 = 0
n2 = 1
n3 = 0

termo = int(input('INFORME UM NÚMERO PARA VER SUA SEQUÊNCIA FIBONACCI: '))
cont = 0

while (cont < termo):
    print(f'{n1}, {n2}, {n3}', end='')
    n1 = n2
    n2 = n3
    n3 = n1 + n2
    cont = cont + 1
